tofhir-server/src/main/scala/io/tofhir/server/model/Project.scala

```diff
@@ -17,7 +17,7 @@ case class Project(id: String = UUID.randomUUID().toString,
                    schemas: Seq[SchemaDefinition] = Seq.empty,
                    mappings: Seq[FhirMappingMetadata] = Seq.empty,
                    contextConceptMaps: Seq[FhirMappingContext] = Seq.empty,
-                   mappingJobs: Seq[FhirMappingJob] = Seq.empty
+                   mappingJobs: Seq[JobMetadata] = Seq.empty
```

Comment:
I think we should not use an implementation-specific model (JobMetadata) inside a general-purpose model (Project)